charts/ia-case-notifications-api/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: ia-case-notifications-api
 home: https://github.com/hmcts/ia-case-notifications-api
-version: 0.0.11
+version: 0.0.13
```

Comment:
Is there a reason we bumped up 2 numbers?

Reply:
I think 0.0.12 was used already in another PR